dts/bindings/dsa/dsa.yaml

```diff
@@ -0,0 +1,26 @@
+# Copyright 2025 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: DSA Device
+
+child-binding:
+  description: Properties of slave port
+
+  include:
+    - name: pinctrl-device.yaml
+    - name: ethernet-controller.yaml
+      property-allowlist:
+        - local-mac-address
```

Comment:
why no ``zephyr,random-mac-address``

Reply:
Let me add the random mac support.
Thanks.